clusterloader2/pkg/prometheus/prometheus.go

```diff
@@ -46,9 +47,33 @@ const (
 	checkPrometheusReadyTimeout  = 15 * time.Minute
 	numK8sClients                = 1
 	nodeExporterPod              = "$GOPATH/src/k8s.io/perf-tests/clusterloader2/pkg/prometheus/manifests/exporters/node-exporter.yaml"
-	nodeExporterTemplateName     = "PROMETHEUS_SCRAPE_NODE_EXPORTER"
 )
 
+var (
+	// DeprecatedEnableServer is a deprecated flag for setting up prometheus server.
+	DeprecatedEnableServer bool
+	// DeprecatedTearDownServer is a deprecated flag for tearing down cluster.
+	DeprecatedTearDownServer bool
+	// EnableServer is a flag for setting up prometheus server.
+	EnableServer bool
+	// TearDownServer is a flag for tearing down cluster.
+	TearDownServer bool
+	// ScrapeEtcd is a flag that enables scraping etcd.
+	ScrapeEtcd bool
+	// ScrapeNodeExporter is a flag that enables scraping node exporter.
+	ScrapeNodeExporter bool
+)
+
+func InitFlags() {
+	// TODO: Remove deprecated flags when all ci-jobs are using new flags.
+	flags.BoolEnvVar(&DeprecatedEnableServer, "enable-prometheus-server", "ENABLE_PROMETHEUS_SERVER", false, "(Deprecated) Whether to set-up the prometheus server in the cluster.")
+	flags.BoolEnvVar(&DeprecatedTearDownServer, "tear-down-prometheus-server", "TEAR_DOWN_PROMETHEUS_SERVER", true, "(Deprecated) Whether to tear-down the prometheus server after tests (if set-up).")
+	flags.BoolEnvVar(&EnableServer, "prometheus-enable-server", "PROMETHEUS_ENABLE_SERVER", false, "Whether to set-up the prometheus server in the cluster.")
```

Comment:
I really don't like it - this is a very common convension to name vars, flags, etc. with "enable-XXX" or "disable-XXX".

Having "prometheus-enables-server" is a bad choice - what we have currently is much better. Please revert this change.

Reply:
Done.